lib/components/narrative/default/default-itinerary.js

```diff
@@ -148,12 +143,36 @@ const ITINERARY_ATTRIBUTES = [
         // FIXME: For CAR mode, walk time considers driving time.
         <>
           <FormattedDuration duration={itinerary.walkTime} />
-          <LegIconWrapper>
+          <LegIconWrapper noSpace>
             <LegIcon leg={leg} size={5} />
           </LegIconWrapper>
         </>
       )
     }
+  },
+  {
+    id: 'carbonEmission',
+    order: 4,
+    render: (itinerary, { co2Config }) => {
+      if(!co2Config) return // If there is no CO2 Config, don't display this element
+
+      const { carbonIntensity, massUnit } = co2Config
+      const co2 = calculateEmissions(carbonIntensity, massUnit, itinerary)
+      return (
+        <>
+          <FormattedNumber
```

Comment:
Should we maybe consider adding some rounding? The nearest 15 or 30 or even 50 might make the values a bit more glancable